pkgs/development/python-modules/reflink/default.nix

```diff
@@ -0,0 +1,31 @@
+{ buildPythonPackage
+, cffi
+, fetchPypi
+, lib
+, pytestCheckHook
+, pytestrunner
+}:
+
+buildPythonPackage rec {
+  pname = "reflink";
+  version = "0.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-ySU1gtskQTv9cDq/wbKkneePMbSQcjnyhumhkpoebjo=";
+  };
+
+  propagatedBuildInputs = [ cffi pytestrunner ];
```

Comment:
pytestrunner is probably only a installInputs.

Reply:
I thought so too, but remove it from `propagatedBuildInputs` and the package doesn't build, for some reason.